Create balenalib base images reference documentation (#3183)
balena-io/docs: pages/reference/base-images/balena-base-images-reference.md

---

> title: Balena base images reference

Reviewer: ```suggestion
title: Deprecated - balenalib base images reference
```

excerpt: Guide to deprecated Docker images published by balena

---

> # {{ $names.company.upper }} base images

Reviewer: To not have search enginees confused, let's be clear about this page too. 

```suggestion
# [Deprecated] balenalib base images
```

> Balenalib was the central home for 26,000+ IoT-focused Docker images built specifically for balenaCloud and balenaOS. Balena ceased publishing updates to balenalib images on June 1, 2025, however, all of the previously published images are still available and still work. 

Reviewer: ```suggestion
Balenalib was the central home for 26,000+ IoT-focused Docker images built specifically for balenaCloud and balenaOS. Balena ceased publishing updates to balenalib images on June 1, 2025, however, all of the previously published images will remain available. 
```

Different dates than previous docs, I recommend not adding date to docs. 
Let the blog handle the date, it doesn't help either in a historical context AFAIK as the docs are concerned.

Author: How about just the year, to give readers some idea?

Reviewer: Sounds good.

Reviewer: Please resolve the conversation once you have implemented a change @alanb128

**They should not be used in any new projects as they will receive no further updates.**

> This page provides historical reference to the previously published base images, as well as crucial information for those looking to replicate one of these images using an up-to-date, officially published image. For a more general guide about choosing a base image that is not tied to our old balenalib library, [see our documentation here](https://docs.balena.io/reference/base-images/balena-base-images/).

Reviewer: ```suggestion
This page provides historical reference to the previously published base images, as well as crucial information for those looking to replicate one of these images using an up-to-date, officially published image. We recommend our [base images documentation](/reference/base-images/balena-base-images/) to choose an alternative base image.
```

Reviewer: Relative URL is important to not reload the image when folks click this link

## Supported Architectures, Distros and Languages

> Balenalib supported the following OS distributions and language stacks: 
> (To see the source for any of the deprecated balenalib base images, [see this repository](https://github.com/balena-io-library/base-images/tree/master/balena-base-images))

Reviewer: ```suggestion
Balenalib supported the following OS distributions and language stacks: 
(Refer to the [balena base images](https://github.com/balena-io-library/base-images/tree/master/balena-base-images) repository to find the Dockerfiles and installer scripts for any of the deprecated balenalib base images.)
```
